src/shared/TeamFinanses/TeamFinanses.tsx

```diff
@@ -0,0 +1,115 @@
+import Modal from '@material-ui/core/Modal';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import React, {useEffect } from 'react';
+
+import axios from 'axios-income';
+
+import classes from './TeamFinanses.module.css';
+
+interface IMember {
+  id: number | string;
+  lp: number | string;
+  name: string;
+  surname: string;
+  cash: number;
+  event: string;
+  importDate: string;
+  team: string;
+  title: string;
+  year: number;
+}
+interface Props {
+  incomes: number | null;
+  currentTeam: string;
+}
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 450,
+  },
+});
+
+const TeamFinanses = ({ incomes, currentTeam } : Props): JSX.Element => {
+
+  const [onePercent, setOnePercent] = React.useState<number>(0);
+  const classe = useStyles();
+
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+
+  function createData(name: string, estate: number, expenses: number) {
```

Comment:
Why not arrow function?

Reply:
i copied this from DOCS. I Changed to arrow function